File: solutions/OreoFlavor/2026-08-10/low/26923.java
import java.util.Scanner;

class Solution {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    int testCase = sc.nextInt();
    
    for(int tc = 1; tc <= testCase; tc++) {
      int days = sc.nextInt();
      int max = 0;
      int min = 1000000;

      for(int i = 0; i < days; i++) {
        int input = sc.nextInt();
        if(max < input) {
          max = input;
        }
        if(min > input) {
          min = input;
        }
      }

      System.out.println("#" + tc + " " + (max - min));
    }
  }
}

File: solutions/HJK9810/2026-08-10/low/26923.py
def algorithm():
    N = int(input())
    visiters = list(map(int, input().split()))

    return max(visiters) - min(visiters)

T = int(input())
# 여러개의 테스트 케이스가 주어지므로, 각각을 처리합니다.
for test_case in range(1, T + 1):
    print(f"#{test_case} {algorithm()}")
